apps/wallet/configs/environment/.env.defaults

```diff
@@ -4,11 +4,14 @@
 API_ENV=devNet
 API_ENDPOINT_LOCAL=http://127.0.0.1:5001/
 API_ENDPOINT_LOCAL_FULLNODE=http://127.0.0.1:9000/
+API_ENDPOINT_LOCAL_FAUCET=http://127.0.0.1:5003/
```

Comment:
The local faucet runs on port `9123`.

Reply:
How do you run it? For me it binds at 5003. It seems to be the default port https://github.com/MystenLabs/sui/blob/f5359589ca1db80dbda1b58ae06568dc2cace3c1/crates/sui-faucet/src/main.rs#L37

Reply:
It runs in the `sui-test-validator` crate.